src/main/kotlin/io/snyk/plugin/Utils.kt

```diff
@@ -198,9 +198,10 @@ fun controlExternalProcessWithProgressIndicator(
     checkCancelled.invoke()
 }
 
-fun isIacEnabled(): Boolean = Registry.`is`("snyk.preview.iac.enabled", false)
+fun isIacEnabled(): Boolean = Registry.`is`("snyk.preview.iac.enabled", pluginSettings().iacScanEnabled)
```

Comment:
we can't do that.
isIacEnabled and iacScanEnabled serve different things. We should not mix them up.
PS for example isIacEnabled  serve to detect if IaC should be shown and if "snyk.preview.iac.enabled" is absent for any reason - IaC will be shown immediately to the GA (as iacScanEnabled  is `true` by default).

Reply:
Which is the desired behaviour - it should be automatically enabled, if no feature flag is present. We deliver the feature-flag with a default value of false in our plugin.xml, so not having it right now should not be an issue. But if/when it's gone, IaC/Container should be enabled by default.